mm2src/mm2_bin_lib/src/mm2_wasm_lib.rs

```diff
@@ -181,7 +181,6 @@ impl From<WasmRpcResponse> for Mm2RpcResponse {
 }
 
 /// The errors can be thrown when using the `mm2_rpc` function incorrectly.
-#[wasm_bindgen]
```

Comment:
nit: we really don't/didn't need that?
looks like it doesn't break compilation or anything.

Reply:
It's conflicting with `impl From<Mm2RpcErr> for JsValue {`. Thinking about it again, I think I should remove `impl From<Mm2RpcErr> for JsValue {`, not `#[wasm_bindgen]`.